DEVELOPERS.md

```diff
@@ -199,4 +199,4 @@ Accord Project documentation files are made available under the [Creative Common
 [jsdoc]: http://usejsdoc.org/
 
 [apache]: https://github.com/accordproject/template-studio-v2/blob/master/LICENSE
-[creativecommons]: http://creativecommons.org/licenses/by/4.0/
\ No newline at end of file
+[creativecommons]: http://creativecommons.org/licenses/by/4.0/
```

Comment:
This can be removed.

Reply:
The change here is unclear to me. But I've done the rest of the revisions.